examples/templates/Example.js

```diff
@@ -95,7 +96,17 @@
             var keys = Object.keys( queryDict );
             for ( var i = 0; i < keys.length; i++ ) {
                 var property = keys[ i ];
-                this._config[ property ] = queryDict[ property ];
+                try {
```

Comment:
The idea is to get typed result from url, here in shadow in order to get numbers in _config directly from url (many shadow url parameter from shadows are numbers)
( content of _config has is used by dat.gui directly for slider/input etc)